package.json

```diff
@@ -1,24 +1,23 @@
 {
-  "name": "@tkxs/tkxs-ui",
-  "version": "1.0.0",
+  "name": "@tkxs/cast-ui",
```

Comment:
I'm curious what the term `cast` is in reference to.

Reply:
Should we change the repo name to match?

Reply:
I believe that's what Design has settled on with the name - from a list of mostly mfg/industrial sounding names ([Design document](https://xd.adobe.com/view/88160b56-513e-4a81-44af-e5309fe8a1f7-fe27/)). I'll confirm that it's settled with Niels/Scott before merging. I do agree that we should change the repo name once confirmed.

Reply:
The name "cast-ui" has been confirmed with Scott, Niels, Jack and Preston.

Reply:
Cool! I'm thinking we might want to do a repo name change coupled with a new PR because `https://github.com/technekes/component-library` is mentioned in many places.